mirage/scenarios/collections.ts

```diff
@@ -133,71 +151,338 @@ export function collectionModerationScenario(server: Server, currentUser: ModelI
- * @param project The project model
- */
-function removedProject(project: ProjectModel) {
-    projectBuilder(project, CollectionSubmissionReviewStates.Removed);
-}
+    projectBuilderArgument.server.create('collection-submission-action', {
+        fromState: CollectionSubmissionReviewStates.InProgress,
+        toState: CollectionSubmissionReviewStates.Pending,
+        actionTrigger: CollectionSubmissionActionTrigger.Submit,
+        target: collectionSubmission,
+        creator: projectBuilderArgument.currentUser,
+        dateCreated: getMinusDate(365),
+    });
 
-/**
- * rejectedProject
- *
- * @description Abstracted function to easily build a rejected project
- * @param project The project model
- */
-function rejectedProject(project: ProjectModel) {
-    projectBuilder(project, CollectionSubmissionReviewStates.Rejected);
+    projectBuilderArgument.server.create('collection-submission-action', {
+        fromState: CollectionSubmissionReviewStates.Pending,
+        toState: CollectionSubmissionReviewStates.Rejected,
+        actionTrigger: CollectionSubmissionActionTrigger.Reject,
+        target: collectionSubmission,
+        comment: 'I do not like your project.',
+        creator: projectBuilderArgument.currentUser,
+        dateCreated: getMinusDate(355),
+    });
+
+    projectBuilderArgument.server.create('collection-submission-action', {
+        fromState: CollectionSubmissionReviewStates.InProgress,
+        toState: CollectionSubmissionReviewStates.Pending,
+        actionTrigger: CollectionSubmissionActionTrigger.Submit,
+        target: collectionSubmission,
+        creator: projectBuilderArgument.currentUser,
+        dateCreated: getMinusDate(345),
+    });
+
+    projectBuilderArgument.server.create('collection-submission-action', {
+        fromState: CollectionSubmissionReviewStates.Pending,
+        toState: CollectionSubmissionReviewStates.Accepted,
+        actionTrigger: CollectionSubmissionActionTrigger.Accept,
+        comment: 'You have really improved your project. Welcome!',
+        target: collectionSubmission,
+        creator: projectBuilderArgument.currentUser,
+        dateCreated: getMinusDate(335),
+    });
+
+    projectBuilderArgument.server.create('collection-submission-action', {
+        fromState: CollectionSubmissionReviewStates.Accepted,
+        toState: CollectionSubmissionReviewStates.Removed,
+        actionTrigger: CollectionSubmissionActionTrigger.AdminRemove,
+        comment: 'I am taking my project private.',
+        target: collectionSubmission,
+        creator: projectBuilderArgument.currentUser,
+        dateCreated: getMinusDate(325),
+    });
+
+    projectBuilderArgument.server.create('collection-submission-action', {
+        fromState: CollectionSubmissionReviewStates.InProgress,
+        toState: CollectionSubmissionReviewStates.Pending,
+        actionTrigger: CollectionSubmissionActionTrigger.Submit,
```

Comment:
Since this was removed earlier, the action attributes should be

```suggestion
        fromState: CollectionSubmissionReviewStates.Removed,
        toState: CollectionSubmissionReviewStates.Pending,
        actionTrigger: CollectionSubmissionActionTrigger.Resubmit,
```

Same for line 394